api/db/repositories/productRepository.js

```diff
@@ -3,18 +3,19 @@ const Prices = require('../dbHelper').Prices;
 
 exports.Products = Products;
 
-exports.createProduct = (productData) => {
-    return new Promise((resolve, reject) => {
-        Products.create(productData, { include: [Products.Price], as: 'personId' })
-            .then((createdProduct) => {
-                resolve(createdProduct);
-            })
-            .catch((error) => {
-                reject(error);
-            });
-    });
+exports.createProduct = async (productData) => {
+    if (productData === null || productData === undefined) {
+        throw new Error('Provide Product object to create new Product.');
+    }
+
+    try {
+        return await Products.create(productData, { include: [Products.Price], as: 'personId'});
+    } catch (error) {
+        throw new Error(error);
+    }
 };
 
+//TODO
```

Comment:
is this comment needed here?

Reply:
if it is then add a description of what has to be done `//TODO: change Promise to async below` for example

Reply:
Roger